TOOLS/lua/autoload.lua

```diff
@@ -67,9 +75,45 @@ function SetUnion (a,b)
     return a
 end
 
-function Split (s)
+-- Returns first and last positions in string or past-to-end indices
+function FindOrPastTheEnd (string, pattern, start_at)
+    local pos1, pos2 = string.find(string, pattern, start_at)
+    return pos1 or #string + 1,
+           pos2 or #string + 1
+end
+
+function Split (list)
```

Comment:
not, function names are inconsistent. should be lower_case probably.

Reply:
I followed the style of original Split (also Set/SetUnion above). Should I make them all lower_case?

Reply:
No, if there is precedence for this style, no need to bother. Can be changed separately.